### PR TITLE
:grin: add basic route

### PR DESCRIPTION
**Pull request by beseven**

### CODE REVIEW

File: app/middleware/renderer.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+function results(req, res) {
+  res.render('results');
+}
+
+function findHelp(req, res) {
```
Comment (neilbmclaughlin) [inline app/middleware/renderer.js]: Should this still be findHelp? Maybe more GP specific?

File: app/views/find-help.nunjucks [hunk 2]
```diff
@@ -0,0 +1,46 @@
+{% extends 'layout.nunjucks' %}
```
Comment (neilbmclaughlin) [inline app/views/find-help.nunjucks]: Is find-help the right name for this file?

File: app/middleware/renderer.js (hunk 1 above)
Comment (beseven) [inline app/middleware/renderer.js]: I've asked Dean to provide input on the urls, so raised as an issue

File: app/views/find-help.nunjucks (hunk 2 above)
Comment (beseven) [inline app/views/find-help.nunjucks]: I've asked Dean to provide input on the urls, so raised as an issue

**Pull request merged**